### ISSUE TITLE
MACOS new unimplemented errors - appear related to 16bit offset jump tables

### ISSUE THREAD

**Issue by gbody** [uxmal/reko#618]
The errors shown below all appear near to PC based 16bit offset jump tables.
Error,001013B6,Unimplemented condition code LE.
Error,0010667E,Unimplemented condition code LE.
Error,00101C64,Unimplemented condition code LE.
Error,0010F190,The method or operation is not implemented.
Error,0010EC12,The method or operation is not implemented.
Error,0010A46C,The method or operation is not implemented.
Error,00103F4E,The method or operation is not implemented.

**Comment by ptomin** [comment]
I have same errors on my private binaries

**Comment by uxmal** [comment]
I have fixes for these but they're not quite ready to be pushed yet.

**Comment by uxmal** [comment]
@ptomin , @gbody : I've made the fixes available on a new `indirect-jumps` branch. There's still some errors in the regression suite on that branch, but I was hoping you could try that branch to see if the errors you are reporting go away?

**Comment by gbody** [comment]
Just realised that I didn't get the branch correctly earlier. Retried indirect jump branch, those errors are gone during binary scanning. 
It also appears to repeat the complete RTL code for each procedure that has the indirect jump.

Tried reconstructing data types and received these warnings.

Warning,0010142A,Expected sizes of arrays to have been determined by now
Warning,00101CA0,Expected sizes of arrays to have been determined by now
Warning,00103F7C,Expected sizes of arrays to have been determined by now
Warning,001066A4,Expected sizes of arrays to have been determined by now
Warning,001087CC,Expected sizes of arrays to have been determined by now
Warning,001087DE,Expected sizes of arrays to have been determined by now
Warning,0010A48C,Expected sizes of arrays to have been determined by now
Warning,0010EC32,Expected sizes of arrays to have been determined by now
Warning,0010F1CA,Expected sizes of arrays to have been determined by now
Warning,0010F1F6,Expected sizes of arrays to have been determined by now
Warning,0010F2F2,Expected sizes of arrays to have been determined by now
Warning,0010F33C,Expected sizes of arrays to have been determined by now
Warning,0010F378,Expected sizes of arrays to have been determined by now

**Comment by uxmal** [comment]
The array size warnings are not material to the scanning phase; you can disregard them for now. Can you give me the address of a procedure that exhibits the repetition you reported above?

**Comment by gbody** [comment]
@uxmal While scrolling through I noticed that the RTL code appeared twice. Examples are Get_Input, Get_Kind.

**Comment by gbody** [comment]
@uxmal I have gone back through older versions and it appears to have been occuring in all the versions I have checked. It appears to occur where there is a jump table in a procedure. It also occurs in a x86 windows programn I looked at. The repeat appears to occur to fill the RTL window while the assembly window is still in the procedure. This occurs when you select a procedure and the display has assembly on the left and RTL on the right. Should the displays scroll and sync by address.
ie if the RTL code is multiple lines for an instruction should the assembly also insert blank lines keep both windows in sync. Currently when window is scrolled the other also scrolls.
